docs/guides/esp32/setup_idf_chip.md

```diff
@@ -57,22 +57,29 @@ step.
 -   [Linux](https://github.com/project-chip/connectedhomeip/blob/master/docs/guides/BUILDING.md#installing-prerequisites-on-linux)
 -   [macOS](https://github.com/project-chip/connectedhomeip/blob/master/docs/guides/BUILDING.md#installing-prerequisites-on-macos)
 
-### Prepare for building
+### Bootstrap Matter environment
 
-Before running any other build command, the `scripts/activate.sh` environment
-setup script should be sourced at the top level. This script takes care of
-downloading GN, ninja, and setting up a Python environment with libraries used
-to build and test.
+Execute the below command to bootstrap Matter environment. This script takes
+care of downloading GN, ninja, and setting up a Python environment with
+libraries used to build and test.
 
 ```
-source scripts/activate.sh
+source scripts/bootstrap.sh
 ```
 
-If this script says the environment is out of date, it can be updated by
-running:
+Whenever Matter environment is out of date, it can be updated by running above
+command.
+
+For MacOS, `gdbgui` python package will not be installed using `bootstrap.sh`
+script as it is restricted only for x64 Linux platforms. It is restricted
+because, building wheels for `gevent` (dependency of `gdbgui`) fails on MacOS.
+
+Workaround is to install `gdbgui` wheels as binary:
 
 ```
-source scripts/bootstrap.sh
+source scripts/activate.sh
```

Comment:
If we just bootstrapped, why do we need to activate?

I assume this step needs to be run after every bootstrap, right?  So perhaps document that after bootstrapping you need to do this part.

Reply:
Oh, I did not know bootstrapping will also activate the environment, removed from here.

Yeah, this part has to be run after every bootstrapping.